tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -44,7 +44,7 @@ export type TextureInfo = {
   texture?: GPUTexture|GPUExternalTexture
 };
 
-type ResourceTensorInfo = {
+type GPUResourceData = {
```

Comment:
Is it better to rename to `TensorData` since it also includes some other information ?